src/triton_cli/parser.py

```diff
@@ -396,49 +363,17 @@ def handle_infer(args: argparse.Namespace):
 # Profile
 # ================================================
 def parse_args_profile(parser):
-    profile = parser.add_parser(
-        "profile", help="Profile LLM models using Perf Analyzer"
-    )
+    profile = parser.add_parser("profile", help="Profile models", add_help=False)
     profile.set_defaults(func=handle_profile)
-    add_model_args([profile])
-    add_profile_args([profile])
-    add_backend_args([profile])
-    add_client_args([profile])
+    profile.add_argument(
+        "--help", action="store_true", help="Show help message and exit"
```

Comment:
Can you show me the `triton -h` and `triton profile -h` outputs after these changes?

Reply:
Good question! Sure.

`triton -h`:
![image](https://github.com/triton-inference-server/triton_cli/assets/58150256/eec269a1-3712-438b-aa79-6a01473b6901)

`triton profile -h` (cuts off at the end, this is the GenAI-Perf help output):
![image](https://github.com/triton-inference-server/triton_cli/assets/58150256/44234583-009d-439d-acac-92b6479100fe)